### PR TITLE
Add more validator address / seed nodes related checks and logs

### PR DESCRIPTION
**Pull request by LukasGasior1**

### CODE REVIEW

File: core/src/main/java/com/radixdlt/p2p/discovery/PeerDiscovery.java [hunk 1]
```diff
@@ -136,6 +136,12 @@ public EventProcessor<DiscoverPeers> discoverPeersEventProcessor() {
       final var seedNodes = seedNodesConfigParser.getResolvedSeedNodes();
       this.addressBook.addUncheckedPeers(seedNodes);
 
+      if (seedNodes.isEmpty() && this.addressBook.knownPeers().isEmpty()) {
+        log.warn(
+            "Warning! No valid seed nodes have been configured for this node."
```
Comment (jakrawcz-rdx) [inline core/src/main/java/com/radixdlt/p2p/discovery/PeerDiscovery.java:141]: "... and no peers in address book" ?
(or the `if` should only check `seedNodes`?)

**Review by jakrawcz-rdx — APPROVED** [review]
As always, I can only recommend `"""` for those long literals (but not strictly required).
AFAIS, it is mostly moving code around and adding a few `if ... then log(...)`, looks safe 👍

File: core/src/main/java/com/radixdlt/p2p/discovery/PeerDiscovery.java (hunk 1 above)
Comment (LukasGasior1) [reply core/src/main/java/com/radixdlt/p2p/discovery/PeerDiscovery.java:141]: I did that on purpose :)
- if there are no seed nodes, but some peers have been added to addr book (e.g. other peers connected to us and address-book-synced), then it's fine - no need to log
- if there are no peers in the address book (yet) BUT there are some valid seed nodes - it's fine too, no need to log 
- if there are no peers in the address book AND none were configured - it's a warning, but the root cause is missing seed nodes (so no need to include "... and no peers in address book")

**Comment by LukasGasior1** [comment]
oh god, how have I missed it? :cold_sweat: thanks @jakrawcz-rdx

**Comment by jakrawcz-rdx** [comment]
Fixed by https://github.com/radixdlt/babylon-node/pull/672

**Pull request merged**